core/src/main/java/org/apache/iceberg/MetadataUpdateParser.java

```diff
@@ -72,6 +75,20 @@ private MetadataUpdateParser() {
   // AddSortOrder
   private static final String SORT_ORDER = "sort-order";
 
+  // SetDefaultSortOrder
+  private static final String SORT_ORDER_ID = "sort-order-id";
```

Comment:
This field is called `order-id` in the spec.

Due to discussion in #4693  , we've decided to align the names `order-id` and `write-order-id` on `sort-order-id` (and `default-sort-order-id`) to match the existing code.